docs/source/integrations/pyspark_integration.md

```diff
@@ -40,7 +40,7 @@ class SparkHooks:
 
         # Initialise the spark session
         spark_session_conf = (
-            SparkSession.builder.appName(context._package_name)
+            SparkSession.builder.appName(context._project_path.name)
```

Comment:
Similar comment I made in https://github.com/kedro-org/kedro-starters/pull/125 : should we use the public property here?

```suggestion
            SparkSession.builder.appName(context.project_path.name)
```

Reply:
Yep; agreed. :)